Formula/bwa.rb

```diff
@@ -16,9 +17,13 @@ class Bwa < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "3d875f0b64143cfb58e44681e47102158b46e0004c408d8fc5f10dc976383a94"
   end
 
+  depends_on "sse2neon" => :build if Hardware::CPU.arch == :arm64
+
   uses_from_macos "zlib"
 
   def install
+    inreplace "ksw.c", "<emmintrin.h>", "<sse2neon.h>" if Hardware::CPU.arch == :arm64
```

Comment:
is there any upstream patch for this?

Reply:
Nope, as I think you've just realised. (Thanks for making a note though.) The project is not very actively maintained, although it's mature. Maybe we can make a note of that PR here, merge as-is, and then remove the `inreplace` once/if the bug-fix gets accepted and a new release made?

Reply:
yeah, if you can submit an upstream PR and add a code comment reference that, that would be great.

Reply:
There are 2 different upstream PRs, though both bundle corresponding library:
- `simde` - lh3/bwa#283
- `sse2neon` - lh3/bwa#344

Reply:
@cho-m Yep. The first one is out-of-date and didn't work for me. The second is very simple and works fine. (Basically what I do with the `inreplace`.)